Resources/Prototypes/_Moffstation/PersonalItems/Wearables/CastleKeeper325/Gumbo/mallardslippers.yml

```
@@ -0,0 +1,23 @@
+# Player: CastleKeeper325 - Character: Gumbo
+- type: entity
+  parent: ClothingShoeSlippersDuck
+  id: PersonalItemGumboSlippers
+  name: mallard slippers
+  description: "Mallards don't normally quack, but these two do!"
```

```suggestion
  description: Mallards don't normally quack, but these two do!
```
Doesnt have to be put in quotes to mark it as a string, and im pretty sure theyl show up in the discription if they are there. Unless that was intended...